test/e2e/apimachinery/validatingadmissionpolicy.go

```diff
@@ -31,11 +31,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/feature"
 	"k8s.io/kubernetes/test/e2e/framework"
 	admissionapi "k8s.io/pod-security-admission/api"
 )
 
-var _ = SIGDescribe("ValidatingAdmissionPolicy [Privileged:ClusterAdmin][Alpha][Feature:ValidatingAdmissionPolicy]", func() {
+var _ = SIGDescribe("ValidatingAdmissionPolicy [Privileged:ClusterAdmin][Alpha]", feature.ValidatingAdmissionPolicy, func() {
```

Comment:
Do we want to allow tests to register tags like `[Privileged:ClusterAdmin]` as plain text? Using `framework.WithLabel` would be better.

If we require it, then we can enforce that the plain text strings don't contain any `[]` and thus catch when developers add those back by mistake.

Reply:
I discussed this with @aojea on chat and the conclusion was to leave this as-is for now to minimize the amount of changes that we do in a single PR.

I can probably start doing some lighter validation for `[]` inside the text and catch the ones that shouldn't be added that way anymore, like `[Alpha]`.

Reply:
yeah, this changes have always a big blast radius and the changes take a long time in be detected so we should be conservative on the rollout of this feature